lib/compute.rb

```diff
@@ -0,0 +1,29 @@
+module Declarations
+  module prefills
+    module Prefillables
+      class MonthlyVatReturnService < ApplicationService
+        SENTRY_OWNER = Constants::SentryOwners::DECLARATIONS
+
+        private
+
+        IMPORTATION_VAT_RATES = [0.2, 0.1, 0.085, 0.055, 0.021, 0.0105].freeze
```

Comment:
Consider extracting `IMPORTATION_VAT_RATES` to a configuration file or a constant in a module if these rates are used in multiple places or are subject to change.